src/pages/settings/templates-and-reminders/TemplatesAndReminders.tsx

```diff
@@ -195,14 +199,30 @@ export function TemplatesAndReminders() {
         </Element>
 
         <Element leftSide={t('body')}>
-          <MarkdownEditor
-            value={templateBody?.body}
-            onChange={(value) =>
-              setTemplateBody(
-                (current) => current && { ...current, body: value }
-              )
-            }
-          />
+          {canChangeEmailTemplate ? (
+            <MarkdownEditor
+              value={templateBody?.body}
+              onChange={(value) =>
+                setTemplateBody(
+                  (current) => current && { ...current, body: value }
+                )
+              }
+            />
+          ) : (
+            <div className="flex flex-col items-start">
+              <span className=" text-gray-500" style={{ fontSize: 12 }}>
+                Email body template can be changed on{' '}
+                <strong>Enterprise/Pro</strong> plan.
+              </span>
```

Comment:
This needs to be replaced with a translatable string.

Reply:
> This needs to be replaced with a translatable string.

Added